pom.xml

```diff
@@ -129,6 +128,13 @@
                         <goals>
                             <goal>compile</goal>
                         </goals>
+                        <configuration>
```

Comment:
Curious about this one: was this change needed to support something new? I may have missed a change to include `generated-sources`?

Reply:
Initially, the changes were made to make the `Java` source code (`SpreadWrapper.class`) compilable as well.
41606b5082b582c61d4a9f4fe9b826484e700b7c

On the other hand, there was a problem with this change method.
https://github.com/FasterXML/jackson-module-kotlin/pull/408#issuecomment-833509340

The final change was made by @dinomite, and I don't know the details.
eb169f89adad7ffc9d771862154b3e9b081dc0bf

Reply:
I had to add these explicit `sourceDirs` to make compilation work after `sourceDirectory` was removed above.

`generated-sources` is where the `PackageVersion.java` ends up and the other two are our actual source (the java path for `SpreadWrapper`)